src/WebJobs.Script/Description/Workers/ScriptInvocationContext.cs

```diff
@@ -20,7 +20,7 @@ public class ScriptInvocationContext
 
         public IEnumerable<KeyValuePair<string, string>> Attributes { get; set; }
 
-        public IEnumerable<(string name, DataType type, object val)> Inputs { get; set; }
+        public IEnumerable<(string Name, DataType Type, object Val)> Inputs { get; set; }
```

Comment:
This might be a breaking change. Is this assembly publicly used?

Reply:
It is a public type exposed from the Microsoft.Azure.WebJobs.Script package, so technically you're right. However in practice, this isn't a library customers are building on, just us. So I think this is OK. We've made similar "breaks" in the past.

Reply:
Yeah, what @mathewc said. I believe this is only consumed by Core Tools via an internal feed, and we typically just absorb any API break there when we update the reference. In this case, though, I'm see no reference to this type anyway, so we likely don't even need to adjust anything downstream.